src/tests/functional/plugin/gpu/shared_tests_instances/single_layer_tests/experimental_detectron_detection_output.cpp

```diff
@@ -0,0 +1,66 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "single_layer_tests/experimental_detectron_detection_output.hpp"
+
+#include <vector>
+
+#include "common_test_utils/ov_tensor_utils.hpp"
+
+using namespace ov::test;
+using namespace ov::test::subgraph;
+
+namespace {
+
+const std::vector<ov::test::ElementType> netPrecisions = {
+    ov::element::Type_t::f16,
+    ov::element::Type_t::f32,
```

Comment:
Please add type i32 and i64 because the kernel supports them.

Reply:
If I do that, the test tries to instantiate the operation with int32/int64 boxes/deltas/scores/image_info, which is not supported, so the test fails with the following message:

```
.../src/tests/functional/shared_test_classes/src/base/ov_subgraph.cpp:94: Failure

.../src/plugins/intel_gpu/src/plugin/program.cpp:354 cldnn program build failed! Node which is about to be added in between two other nodes should not have any existing dependencies
```

Shall I investigate?

Reply:
I think input precision of the op is any floating type as document. So f32/f16 seems to be ok for the single layer test.